src/main/kotlin/org/rust/ide/annotator/fixes/AddStructFieldsFix.kt

```diff
@@ -105,4 +109,47 @@ class AddStructFieldsFix(
 
         return null
     }
+
+    private val Ty.defaultValue: String
+        get() {
+            return when (this) {
+                is TyBool -> "false"
+                is TyInteger -> "0"
+                is TyFloat -> "0.0"
+                is TyStr -> "\"\""
+                is TyStruct -> {
+                    when (this.item.name) {
+                        //FIXME: should be check by absolute name e.g "std::string::String"
+                        "String" -> "String::from(\"\")"
+                        else -> "()"
+                    }
+                }
+                is TyEnum -> {
+                    when (this.item.name) {
+                        //FIXME: should be check by absolute name e.g "std::option::Option"
+                        "Option" -> "None"
+                        else -> "()"
+                    }
+                }
+                is TyArray -> {
+                    val size = this.size ?: 0
+                    val elementDefault = this.base.defaultValue
+                    val accum = mutableListOf<String>()
+                    //Kotlin array generator's size is only allowed Int, But TyArray's size is Long.
```

Comment:
Hm, what if the type is `[i8; 256]`? I don't think we should insert 256 values. Let's use `[]`? Though it has a wrong type, I think it's a reasonable default nonetheless.